editor/src/components/editor/editor-component.tsx

```diff
@@ -318,6 +318,7 @@ export const EditorComponentInner = React.memo((props: EditorProps) => {
     }
     if (projectId != null) {
       pushProjectURLToBrowserHistory(projectId, projectName)
+      ;(window as any).utopiaProjectID = projectId
```

Comment:
I added this because `getProjectID` reads the project from here, and that is used in `resolveUsers`